package.json

```diff
@@ -37,11 +37,15 @@
   },
   "devDependencies": {
     "chai": "^3.5.0",
-    "enzyme": "2.9.1",
+    "enzyme": "^3.2.0",
+    "enzyme-adapter-react-14": "^1.0.5",
+    "enzyme-adapter-react-15.4": "^1.0.5",
+    "enzyme-adapter-react-16": "^1.1.1",
```

Comment:
Do we need all these adapters? I believe 16 will be the only one used:

https://github.com/airbnb/enzyme/blob/4b1867e122098fc71078b482be176960957a9be8/packages/enzyme-adapter-react-helper/src/index.js#L12

Reply:
The way I understood it was that `enzyme-adapter-react-16` only works on React 16, `enzyme-adapter-react-15.4` on 15.4, etc. If we want to have i18n-calypso to work with all of the enzyme supported versions, we have to have all of them available, no?

Reply:
I was trying to wrap my head around that, but I wouldn't expect dependents to be running tests.

For local development, I'd expect expect `react@^16.0.0` to be installed and the corresponding adapter being used:

https://github.com/Automattic/i18n-calypso/blob/470a021188daeac1ee105ad91e79ca9a4db9d113/package.json#L35

I'm digging into the dependencies a bit further…